src/components/ble/weather/WeatherService.cpp

```diff
@@ -66,6 +66,14 @@ namespace Pinetime {
           CleanUpQcbor(&decodeContext);
           return BLE_ATT_ERR_INVALID_ATTR_VALUE_LEN;
         }
+
+        // Do not add already expired events
```

Comment:
Even though it would be a valid approach to be more defensive here, I would really really rather have the companions not to write :hankey: to the timeline at all.

Reply:
Its kinda cheap flashwise and I figured, why not early abort here :^)